src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -61,12 +62,27 @@ public class Drivetrain extends SubsystemBase {
           CANMappings.REAR_RIGHT_TURNING,
           DrivetrainConfig.BACK_RIGHT_CHASSIS_ANGULAR_OFFSET);
 
-  public final PIDController orbitDistanceController =
-      new PIDController(
-          OrbitConfig.ORBIT_DISTANCE_P, OrbitConfig.ORBIT_DISTANCE_I, OrbitConfig.ORBIT_DISTANCE_D);
-  public final PIDController orbitRotationController =
-      new PIDController(
-          OrbitConfig.ORBIT_ROTATION_P, OrbitConfig.ORBIT_ROTATION_I, OrbitConfig.ORBIT_ROTATION_D);
+  private TrapezoidProfile.Constraints distanceControllerConstraints =
```

Comment:
This can be cleaned up by creating the constrains inline when creating the controller, instead of it's own field